### PR TITLE
Fix danger zone subheader taking full width in medium and large screen sizes

### PR DESCRIPTION
**Pull request by pavinduLakshan**
### Purpose

$subject

Macbook Pro

Screenshot 2023-11-02 at 11 10 51

Extended Monitor

Screenshot 2023-11-02 at 11 13 47

### Related Issues
- https://github.com/wso2/product-is/issues/17171

### Related PRs
- N/A

### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Comment by Achintha444** [comment]
Add changeset

**Comment by pavinduLakshan** [comment]
> Add changeset

Addressed with 84998e9

File: .changeset/gorgeous-dolls-occur.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/theme": patch
```
Comment (JayaShakthi97) [inline .changeset/gorgeous-dolls-occur.md:2]: Add console, myaccount, identity-apps-core as well.

Comment (pavinduLakshan) [reply .changeset/gorgeous-dolls-occur.md]: Fixed with 6a46abd

**Review by JayaShakthi97 — APPROVED** [review]

**Pull request merged**